rules/ethtool.mk

```diff
@@ -0,0 +1,12 @@
+# ethtool
+
+ETHTOOL_VERSION_BASE = 5.8
+export ETHTOOL_VERSION_BASE
+
+ETHTOOL = ethtool
+$(ETHTOOL)_SRC_PATH = $(SRC_PATH)/$(ETHTOOL)
+$(ETHTOOL)_PATH = $(BUSTER_FILES_PATH)/$(ETHTOOL)
```

Comment:
@shlomibitton this is only valid when you copying files - please remove it

Reply:
Fixed.